compose/service.py

```diff
@@ -523,6 +530,8 @@ def _get_container_create_options(
             for k in DOCKER_CONFIG_KEYS if k in self.options)
         container_options.update(override_options)
 
+        container_options.update(self.get_logging_options())
+
```

Comment:
Why is this required? I thought we only had to provide it to the HostConfig section which is done below.